src/server/tasks/sendmail.js

```diff
@@ -0,0 +1,31 @@
+import Queue from 'bull';
+
+import processor from '~/server/tasks/processor';
+import { isDev } from '~/common/utils/constants';
+import { redisUrl, redisLongRunningOptions } from '~/server/services/redis';
+import mailer, {
+  prodTransporter,
+  devTransporter,
+} from '~/server/services/mailer';
+import submitJob from '~/server/tasks/submitJob';
+
+const mailing = new Queue('Send mails', redisUrl, {
+  settings: redisLongRunningOptions,
+});
+
+processor(mailing).process(
+  async ({ data: { to, subject, template, data = {} } }) => {
+    const send = mailer(isDev ? devTransporter : prodTransporter);
+    return send(to, subject, template, data);
+  },
+);
+
+export const testEmail = (to = 'me@example.org', data = { name: 'Meesix' }) =>
+  submitJob(mailing, {
```

Comment:
This needs a second arg for the job id! Right now it logs like `[[object Object]] ACTIVE "Send mails" job started` and can't find the data object

Reply:
Ohh thanks for the catch. Totally overlooked that one. I think for the test email it might be enough to use the recipient's email address as id. But I wonder if for other mailing tasks this will be sufficient. Is it a possibility that the queue contains more than one legitimate email for one recipient at a time?

Reply:
Maybe ... ideally not, but I think combining it with something else for the real job would be good